### ISSUE TITLE
JULIA_PKG_SERVER and the whole Pkg Server protocol needs docs

### ISSUE THREAD

**Issue by KristofferC** [JuliaLang/Pkg.jl#1996]
It looks like it is completely undocumented.

**Comment by johnnychen94** [comment]
Please feel free to use information from #1876 or just close that one.

**Comment by KristofferC** [comment]
Thanks, good reference. I had missed that.

**Comment by StefanKarpinski** [comment]
Fair enough. I should write this up beyond just linking to the design issue.

**Comment by KristofferC** [comment]
Bump :)

**Comment by StefanKarpinski** [comment]
As soon as I'm not actively working on urgent features for 1.6, I can take some time to document stuff.

**Comment by KristofferC** [comment]
Bump :)

**Comment by KristofferC** [comment]
Bump :)

(would be good to have docs for how to change the server when for example the default server goes down, like EU is now).

**Comment by Arkoniak** [comment]
It would be really nice to have docs (I've asked this question on discourse https://discourse.julialang.org/t/julia-pkg-server-related-documentation/62117 and was redirected here).

**Comment by anandijain** [comment]
Bump again. I really don't understand the relationship between registries and package servers.

I hit a case where I'd like to use two private registries (alongside General). The setup documentation for both of these registries includes setting JULIA_PKG_SERVER. I see that I can't use two URLs here currently, and it seems like this is a bad idea (https://github.com/JuliaLang/Pkg.jl/issues/2507). 

Maybe this isn't exactly on-topic for this issue, but is there a way to use multiple private registries without needing to use their respective package servers?

**Comment by StefanKarpinski** [comment]
Just clone them into `~/.julia/registries`. Doesn't involve pkg server protocol at all.

**Comment by KristofferC** [comment]
Closed by https://github.com/JuliaLang/Pkg.jl/pull/4234
